virttest/utils_v2v.py

```diff
@@ -1615,6 +1702,27 @@ def v2v_supported_option(opt_str):
     return False
 
 
+def is_option_in_v2v_cmd(v2v_cmd, option):
+    """
+    Check if specific v2v option(s) is(are) in v2v command line
+
+    :param v2v_cmd: A v2v command line string
+    :param option: option or option list
+    """
+    if not option or not v2v_cmd:
+        return False
+
+    if isinstance(option, list):
```

Comment:
`return all([opt_i in v2v_cmd for opt_i in option])`

And if v2v_cmd could be made list, I guess there's a simple way to determine whether or not a list is a sub-list of another list.
I mean 'set'

Reply:
Yes, Thanks for your suggestion. I'll update it.

Reply:
:handshake: